resourcekeys/const.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package resourcekeys contains well known type and label keys for resources.
+package resourcekeys
+
+// Constants for Kubernetes resources.
+const (
+	K8STypeContainer = "k8s.io/container"
+
+	K8SKeyClusterName   = "k8s.io/cluster/name"
```

Comment:
Since this is not an official k8s terminology please comment what it is and why it is here.